README.md

```diff
@@ -0,0 +1,3 @@
+npm i
+npm start
+localhost:8080
```

Comment:
may be `3000`?

Reply:
describe here that for `development` we should do `npm start` and open `localhost:3000`, and for `production` we heed to `npm run build` and start node web server

Reply:
looks like node web-server will not work any more..